src/unix/mod.rs

```diff
@@ -228,6 +228,11 @@ cfg_if! {
         #[link(name = "root")]
         #[link(name = "network")]
         extern {}
+    } else if #[cfg(target_os = "fuchsia")] {
+        #[link(name = "c")]
+        #[link(name = "mxio")]
+        #[link(name = "unwind")]
```

Comment:
I think `unwind` actually comes in at the libstd layer rather than the libc layer (e.g. those symbols aren't referenced from this library). Other than that though this looks fine to me!